Firebase/InAppMessaging/Public/FIRInAppMessagingRendering.h

```diff
@@ -104,6 +120,77 @@ NS_SWIFT_NAME(InAppMessagingDisplayMessage)
                       triggerType:(FIRInAppMessagingDisplayTriggerType)triggerType;
 @end
 
+NS_SWIFT_NAME(InAppMessagingCardDisplay)
```

Comment:
This should not be in the public header.

Reply:
Hmm.. not sure what you mean here-- the Swift method name declaration shouldn't be?

Reply:
There was a question about whether the `FIRInAppmessagingCardDisplay`class should be public - I don't recall it being part of the API surface before from the proposal.

Reply:
Ah, sorry for the confusion. We need these message subclasses to be public so developers can inspect them in the callbacks. F/e if a developer implements `impressionDetectedForMessage:` they wouldn't have access to the metadata (title, body, etc.) for the message unless they do something like:

```
override func impressionDetected(for inAppMessage: InAppMessagingDisplayMessage) {
    guard let message = inAppMessage as? InAppMessagingCardDisplay else {
        return
    }
    if message.title = "Discount swim lessons" {
        // Do something.
    }
}
```

Reply:
Addressed this by putting the initializer in a private header in 1483cf6.